include/dev/SystemDetect.hpp

```diff
@@ -32,7 +32,7 @@ class SystemDetect {
     };
 
     /**
-     * Create the system detect device which will work to indetify the
+     * Create the system detect device which will work to indentify the
      * provided heart beat CANopen IDs.
      *
      * @param[in] bikeHeartBeat The heart beat CANopen ID associated with the
```

Comment:
```suggestion
     * @param[in] bikeHeartBeat The heartbeat CANopen ID associated with the
```
suggestion to fix in all areas where "heart beat" is written